src/NServiceBus.Core/Transports/Msmq/QueuePermissions.cs

```diff
@@ -55,22 +72,9 @@ static void WarnIfPublicAccess(MessageQueue queue)
             }
         }
 
-        public static void SetPermissionsForQueue(MessageQueue queue, string account)
-        {
-
-            queue.SetPermissions(LocalAdministratorsGroupName, MessageQueueAccessRights.FullControl, AccessControlEntryType.Allow);
-            queue.SetPermissions(LocalEveryoneGroupName, MessageQueueAccessRights.WriteMessage, AccessControlEntryType.Allow);
-            queue.SetPermissions(LocalAnonymousLogonName, MessageQueueAccessRights.WriteMessage, AccessControlEntryType.Allow);
-
-            queue.SetPermissions(account, MessageQueueAccessRights.WriteMessage, AccessControlEntryType.Allow);
```

Comment:
I've added a test that shows that MSMQ by default only gives write permissions by default:

https://github.com/Particular/NServiceBus/pull/4304/commits/162b56e239f9e1507cb18017ecb6a5133511f8bb

so I think we need to keep this for now to give receive and peek as well?